### DISCUSSION TITLE
Contribution request: Add Icon to walkxcode/dashboard-icons

### DISCUSSION THREAD

**Discussion by eJonny** [netzero-labs/netzero#6] [Ideas]
walkxcode/dashboard-icons is a relatively well know repository for icons. Any objection if I contribute the netzero icon to that repo so it can be easily used in dashboard applications?

**Comment by zigam** [comment]
Sounds good, thanks!

**Reply by eJonny** [reply]
Ok. I'll close the discussion when PR is approved in that repo.
